defaults/main.yml

```diff
@@ -1,5 +1,9 @@
 clamav_cron_frequency: weekly
 
+apparmor_clamd_configuration_path: /etc/apparmor.d/usr.sbin.clamd
+apparmor_freshclam_configuration_path: /etc/apparmor.d/usr.bin.freshclam
+apparmor_complain: true
```

Comment:
`apparmor_complain`, at least, needs to be specified as a role variable in `README.md`.